File: ApplicationCore/Entities/Provider.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace ApplicationCore.Entities
{
    public class Provider : BaseEntity
    {
        [Required]
        public string Name { get; set; }
        [Required]
        public string Email { get; set; }
        [Required]
        public DateTime TimeWorkWith { get; set; }
        [Required]
        public DateTime TimeWorkTo { get; set; }
        [Required]
        public bool IsActive { get; set; }
        public string Path { get; set; }
        public string WorkingDays { get; set; }
        [Required]
        public bool IsFavorite { get; set; }
        public string Info { get; set; }
        public ICollection<Menu> Menus { get; set; }
    }
}

File: ClientApp/src/app/provider/provider.ts
import { Data } from '@angular/router';

export class Provider{
    public id:number;
    public name:string;
    public email:string;
    public timeWorkWith:Data;
    public timeWorkTo:Data; 
    public isActive:boolean;
    public path:File;
    public workingDays:string;   
    public isFavorite:boolean;
    public info:string;
}